ext/devcontainer/src/azd/README.md

```diff
@@ -0,0 +1,25 @@
+# Azure Developer CLI (azd)
```

Comment:
> Looks like you want to use the devcontainers/action@v1 action to automatically generate docs (ie README.md)? However, you are missing some steps which helps open PRs for doc updates.

See https://github.com/devcontainers/feature-starter/blob/main/.github/workflows/release.yaml#L26-L47

FYI: This file would be auto-generated as you have `generate-docs: "true"` and once you follow my previous comment.